html_app/instructor.js

```diff
@@ -0,0 +1,65 @@
+
+$(function(){
+
+    $('.scb_s_dashboard_table_row').hover(function(){
+	    $('.scb_s_dashboard_link', this).toggle();
+    });
+    /**
+     * Course setup
+     */
+    $('.scb_f_course_setup_create_new_course_option input').click(function(){
```

Comment:
This would not work because if I click on the same element twice it toggle twice and make the text visible.